examples/ray/data/rag/ray-data-pipeline/.env.example

```
@@ -0,0 +1,43 @@
+MILVUS_TEXT_MAX_CHARS=8192
+
+# Collection management
+DROP_EXISTING_COLLECTION=true
+
+# vLLM embedding (GPU)
+VLLM_MODEL_SOURCE=intfloat/multilingual-e5-large
+VLLM_BATCH_SIZE=4
+VLLM_CONCURRENCY=1
+EMBEDDING_DIM=1024
+# VLLM_ACCELERATOR_TYPE=NVIDIA-A10G
+VLLM_ENGINE_KWARGS_JSON={"enforce_eager": true, "gpu_memory_utilization": 0.8}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VLLM_ENGINE_KWARGS_JSON key should go before the VLLM_MODEL_SOURCE key

(UnorderedKey)